error-503.html

```diff
@@ -7,125 +7,159 @@
+        </ul>
+        <ul class="main-nav__account-menu account-menu">
+          <li><a href="https://www.nhs.uk/Personalisation/Login.aspx">Log in</a></li>
+          <li><a href="https://www.nhs.uk/Personalisation/Registration.aspx">Create an account</a></li>
+        </ul>
       </div>
-    </header>
+    </nav>
+    <!-- endnoindex -->
+  </header>
+
+
+      <main id="content" role="main" class="util-nofocus">
```

Comment:
Is the role attribute needed here?
![screen shot 2017-12-04 at 15 49 15](https://user-images.githubusercontent.com/639706/33561604-0490bc7c-d90b-11e7-9e89-d6de27e7d443.png)

Reply:
Nope, not if the element is `<main>` well spotted.